### PR TITLE
Updates to support custom extensions in codemod-cli

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
Pending: https://github.com/rwjblue/codemod-cli/pull/62

### CODE REVIEW

**Comment by rwjblue** [comment]
https://github.com/rwjblue/codemod-cli/pull/62 is merged and released, should be ready to update...

File: bin/cli.js [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
 const args = process.argv.slice(3);
 
-console.log(transformName, args);
+console.log(args);
 
-require('codemod-cli').runTransform(__dirname, transformName, args);
+require('codemod-cli').runTransform(
+    __dirname, 
+    transformName, 
+    extensionArgs.concat(args)
```
Comment (rwjblue) [inline bin/cli.js]: This needs to be tweaked still:

```suggestion
    args,
    'hbs'
```

File: bin/cli.js [hunk 2]
```diff
@@ -1,9 +1,15 @@
 #!/usr/bin/env node
 'use strict';
 
+
+const extensionArgs = ['--extensions', 'hbs'];
```
Comment (rwjblue) [inline bin/cli.js]:
```suggestion
```

Comment (NullVoxPopuli) [reply bin/cli.js]: this broke things :-\

Comment (NullVoxPopuli) [reply bin/cli.js]: I think because previously, the file path was the last thing, and now the extension is. The path needs to be last :-\

Comment (NullVoxPopuli) [reply bin/cli.js]: or, I'm pulled to thin, and don't know how this works:
`  await execa('../../../bin/cli.js', ['http://localhost:4200', 'app'], execOpts);`

Comment (NullVoxPopuli) [reply bin/cli.js]: anyway, to make this PR worth merging, the integration tests should pass *

Comment (rwjblue) [reply bin/cli.js]: > The path needs to be last :-\

Ya, not sure why you think that. `runTransform` is defined like:

```js
function runTransform(binRoot, transformName, args, extensions = DEFAULT_EXTENSIONS) {
```

So the paths aren't last, they are the third argument, and the extensions (comma separated string) is last.

Comment (rwjblue) [reply bin/cli.js]: I'll poke at the branch though, maybe I made a mistake in the impl

Comment (NullVoxPopuli) [reply bin/cli.js]: > Ya, not sure why you think that. runTransform is defined like:

legit! I did not check that out. I think my assumption was a red herring.

**Comment by NullVoxPopuli** [comment]
@rwjblue integrate tests pass now :) 
(not that they do anything yet, but we can do another PR to add real world:tm: stuff)

**Pull request merged**